### PR TITLE
Uniform exec calls

### PR DESCRIPTION
**Pull request by iobuhov**
## This PR contains

-   [ ] Bug fix
-   [ ] Feature
-   [x] Refactor
-   [ ] Documentation
-   [ ] Other (describe)

### CODE REVIEW

File: packages/tools/release-utils-internal/src/steps.ts [hunk 1]
```diff
@@ -12,18 +12,16 @@ export async function updateChangelogsAndCreatePR(
     const releaseBranchName = `${releaseTag}-update-changelog`;
 
     console.log(`Creating branch '${releaseBranchName}'...`);
-    await execShellCommand(`git checkout -b ${releaseBranchName}`);
+    await exec(`git checkout -b ${releaseBranchName}`, { stdio: "inherit" });
```
Comment (r0b1n) [inline packages/tools/release-utils-internal/src/steps.ts:15]: `{ stdio: "inherit" }` is repeated quite some times, can we reverse it, make it a default option and use additional options in places where we need output to be returned?

File: packages/tools/release-utils-internal/src/steps.ts [hunk 2]
```diff
@@ -12,18 +12,16 @@ export async function updateChangelogsAndCreatePR(
...
 
     console.log(`Committing CHANGELOG.md to '${releaseBranchName}' and pushing to remote...`);
-    await execShellCommand([
-        `git add ${changelog.changelogPath}`,
-        `git commit -m "chore(${packageInfo.packageName}): update changelog"`,
-        `git push ${remoteName} ${releaseBranchName}`
-    ]);
+    await exec(`git add ${changelog.changelogPath}`);
+    await exec(`git commit -m "chore(${packageInfo.packageName}): update changelog"`);
+    await exec(`git push ${remoteName} ${releaseBranchName}`);
```
Comment (r0b1n) [inline packages/tools/release-utils-internal/src/steps.ts:24]: `{ stdio: "inherit" }` is probably missing for those (if we don't want to change it to the default option).

Comment (iobuhov) [reply packages/tools/release-utils-internal/src/steps.ts:24]: Done

File: packages/tools/release-utils-internal/src/steps.ts (hunk 1 above)
Comment (iobuhov) [reply packages/tools/release-utils-internal/src/steps.ts]: Done

File: packages/tools/release-utils-internal/src/mpk.ts [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
+    const existingImages = (await exec(`docker image ls -q mxbuild:${version}`)).stdout.trim();
     if (!existingImages) {
         console.log(`Creating new mxbuild:${version} docker image...`);
         const dockerfilePath = join(process.cwd(), "packages/tools/pluggable-widgets-tools/scripts/mxbuild.Dockerfile");
-        await execShellCommand(
+        await exec(
             `docker build -f ${dockerfilePath} ` +
                 `--build-arg MENDIX_VERSION=${version} ` +
-                `-t mxbuild:${version} ${process.cwd()}`
+                `mxbuild:${version} ${process.cwd()}`
```
Comment (r0b1n) [inline packages/tools/release-utils-internal/src/mpk.ts:19]: `-t` (tag) is gone, is it intentional?

Comment (iobuhov) [reply packages/tools/release-utils-internal/src/mpk.ts:19]: This is intentional, we don't need terminal to build image

Comment (r0b1n) [reply packages/tools/release-utils-internal/src/mpk.ts:19]: This is true for `-t` in `docker run` commands. For `docker build` command this means [tag an image](https://docs.docker.com/engine/reference/commandline/build/#tag-an-image--t) 🙈

Comment (iobuhov) [reply packages/tools/release-utils-internal/src/mpk.ts:19]: No waaaay 🤨 😕

Comment (iobuhov) [reply packages/tools/release-utils-internal/src/mpk.ts]: Done

**Pull request merged**